zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -614,7 +614,7 @@ where
                 _ => unreachable!("unmatched response to a solution rate request"),
             };
 
-            Ok(solution_rate)
+            Ok(solution_rate as u64)
```

Comment:
In general we avoid `as`, because it does silently wrapping integer casts, which can hide bugs.

Overflows are almost impossible in this situation, so we can do:
```suggestion
            Ok(solution_rate.try_into().expect("per-second solution rate always fits in u64"))
```

Reply:
(Otherwise we'd add a comment explaining why the cast is ok, but we try to implement those kinds of comments as `expect()`, so the code checks them.)